### ISSUE TITLE
Add Twitter handle(s) is streamer(s) and optional description

### ISSUE THREAD

**Issue by Sabonzy** [christophrumpel/larastreamers#33]
I think it will be nice when importing stream you ask them to provide the Twitter handle(s) of the streamer(s) and an optional description. This will be useful when tweeting out about the stream on Twitter and people will know beforehand whose stream it is

eg: 
![image](https://user-images.githubusercontent.com/8821242/118616355-23a45a00-b7b1-11eb-8067-61a2a46cd716.png)

or 
![image](https://user-images.githubusercontent.com/8821242/118616599-5e0df700-b7b1-11eb-8e4d-9a3cf6d07781.png)

**Comment by christophrumpel** [comment]
That's true. Would be cool if Twitter was connected to the YouTube videos, but it is not. But I have another idea:

@nuernbergerA what about the attach a Twitter handle to channels that we add? This way we could check if the video has a connected channel, and if it does we show a Twitter handle on the homepage's list, but also add it to the automatic tweets.
